htdocs/product/inventory/card.php

```diff
@@ -80,8 +82,8 @@
 {
 	$error=0;
 
-	$permissiontoadd = $user->rights->stock->creer;
-	$permissiontodelete = $user->rights->stock->supprimer;
+	$permissiontoadd = $user->rights->stock->advance_inventory->create;
```

Comment:
If the feature "manage advanced permission" is not on, what is the permission to add or delete ?
Currently it should be permission to add/update stock.

Reply:
I just use the perms define into modStock